nns/nns_contract.go

```diff
@@ -374,16 +375,31 @@ func Register(name string, owner interop.Hash160, email string, refresh, retry,
 	return true
 }
 
-func saveCommitteeDomain(ctx storage.Context, name, email string) {
-	putTLD(ctx, makeTLDKey(name))
+// RegisterTLD registers new top-level domain. RegisterTLD MUST be called by the
+// committee only. Name MUST be a valid TLD.
+//
+// RegisterTLD panics with 'TLD already exists' if domain already exists.
+func RegisterTLD(name, email string, refresh, retry, expire, ttl int) {
+	checkCommittee()
+	saveCommitteeDomain(storage.GetContext(), name, email, refresh, retry, expire, ttl)
+}
+
+// saveCommitteeDomain marks TLD as registered via prefixRoot<name> storage
+// record and saves domain state calling saveDomain with given parameters and
+// empty owner. The name MUST be a valid TLD name.
+func saveCommitteeDomain(ctx storage.Context, name, email string, refresh, retry, expire, ttl int) {
```

Comment:
This method isn't used anywhere else, thus please, move this code directly into RegisterTLD. NNS has a lot of tiny methods-helpers, and it's already hard to distinguish them, don't make it harder.

Reply:
it's called in `RegisterTLD` and `_deploy`, just wanted to share similar instructions. But i don't mind to unwrap it, do u think it's worth it?